flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1869,4 +1908,16 @@ public boolean isForce() {
 			return props.forceCheckpoint();
 		}
 	}
+
+	private enum CoordinatorRestore {
```

Comment:
nit: the argument of this type is named `coordinatorRestoreBehavior` above. I think it makes sense to add "Behavior" here too.